plugins/autoloader/prism-autoloader.js

```diff
@@ -152,6 +152,13 @@
 		"yml": "yaml"
 	}/*]*/;
 
+	/**
+	 * @typedef LangDataItem
+	 * @property {{ success: function, error: function }[]} callbacks
```

Comment:
The way the code checks them below, each of these are optional?

Reply:
Yes, they are!